contrib/appengine-standard-util/src/main/proto/trace_id.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.contrib.appengine.standard.util";
+option java_outer_classname = "TraceProto";
+
+message TraceIdProto {
```

Comment:
Why do we need to define a proto?  If CloudTraceContext uses a protocol buffer format for its trace ID, does it have a trace ID proto that we can reuse?

Reply:
No public proto. See also https://github.com/GoogleCloudPlatform/cloud-trace-java/tree/master/services/appengine-standard-service/src/main/proto/cloud/trace/proto